src/System.ComponentModel.TypeConverter/src/System/ComponentModel/AttributeCollection.cs

```diff
@@ -378,6 +378,19 @@ object ICollection.SyncRoot
             }
         }
 
+        int ICollection.Count 
```

Comment:
I didn't double check but if these are truly calling all the implicit members then I agree they are redundant and aren't actually needed for compat. When I generated what is in netstandard I took exactly what was in desktop and they do appear to implement some of the members both implicitly and explicitly, my best guess is that we did it one way in one version and then added the other version in a later version (crossing my fingers that there is a real reason :)). 

With that said I'm not going to complain either way as long as the API surface is compatible with .NET Standard.